projects/ngrx.io/content/guide/migration/v16.md

```diff
@@ -154,6 +154,129 @@ import { createMockStore } from '@ngrx/store/testing';
 const mockStore = createMockStore();
 ```
 
+#### LetModule
```

Comment:
Deprecations need to be added as a separate section. Move added lines to the bottom of this file and add the following headings:

```md
## Deprecations

### @ngrx/component

#### LetModule

....

#### PushModule

...
```

Reply:
Oops, yeah, you're right!